fineract-core/src/main/java/org/apache/fineract/infrastructure/core/config/Resilience4JProperties.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.fineract.infrastructure.core.config;
+
+import io.github.resilience4j.common.retry.configuration.CommonRetryConfigurationProperties;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Getter
+@Setter
+@ConfigurationProperties(prefix = "resilience4j")
+public class Resilience4JProperties {
```

Comment:
Is this intended to overlap with Resilience4j's "namespace"? If these are Fineract specific properties applied to Resilience4j then I'd prefix this rather with "fineract.resilience4j"

Reply:
yes, it was intended. Priorly Retry annotation was used with a specific name and it was loading the configurations from the application properties. I had to change to use programatic approach to be able to have more fine-grained configuration. 

I would be hesitant to use `fineract.resilience4j.*` for programatic retry logic and `resilience4j.*` for declarative.

Thoughts?

Reply:
You convinced me :)